### PR TITLE
Add support for 'multipart/form-data'

### PR DESCRIPTION
**Pull request by glimchb**
- parse content type
- extract boundary
- extract 3 redfish spec parts (UpdateParameters, UpdateFile, OemXXX)
- save data to temp file
- return success

Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

**Comment by glimchb** [comment]
log example:
```
redfish-mockup-server-bmc-1  |    POST: Headers: Host: 127.0.0.1:8000
redfish-mockup-server-bmc-1  | User-Agent: python-requests/2.27.1
redfish-mockup-server-bmc-1  | Accept-Encoding: gzip, deflate
redfish-mockup-server-bmc-1  | Accept: */*
redfish-mockup-server-bmc-1  | Connection: keep-alive
redfish-mockup-server-bmc-1  | Content-Type: multipart/form-data; boundary=743667863fab4c9893ba103acb6e2b66
redfish-mockup-server-bmc-1  | X-Auth-Token: 1234567890ABCDEF
redfish-mockup-server-bmc-1  | OData-Version: 4.0
redfish-mockup-server-bmc-1  | Content-Length: 67106315
redfish-mockup-server-bmc-1  |
redfish-mockup-server-bmc-1  |
redfish-mockup-server-bmc-1  |    POST: Content-Type: multipart/form-data and {'boundary': '743667863fab4c9893ba103acb6e2b66'}
redfish-mockup-server-bmc-1  |    POST: MULTIPART: name=UpdateParameters and file=None
redfish-mockup-server-bmc-1  |    POST: MULTIPART: name=UpdateFile and file=my-fw-file-name-ver-123.bin
redfish-mockup-server-bmc-1  |    POST: Data: {}
redfish-mockup-server-bmc-1  | TBD: handle multipart
redfish-mockup-server-bmc-1  | 172.25.0.1 - - [09/Apr/2024 19:47:19] "POST /redfish/v1/UpdateService/update-multipart HTTP/1.1" 204 -
```

@MoshiBin @mraineri fyi

File: redfishMockupServer.py [hunk 1]
```diff
@@ -701,6 +730,10 @@ def do_POST(self):
                 elif "TelemetryService/Actions/TelemetryService.SubmitTestMetricReport" in self.path:
                     r_code = self.handle_telemetry(data_received)
                     self.send_response(r_code)
+                # UpdateService (SimpleUpdate is handled part of the Actions, so only multipart is here)
+                elif "UpdateService/update-multipart" in self.path:
+                    logger.info("TBD: handle multipart")
```
Comment (mraineri) [inline redfishMockupServer.py:735]: Is this meant to start a task based on the issue filed for this support?

Comment (glimchb) [reply redfishMockupServer.py:735]: Yes, this should start a new task
Should be in separate pull request

**Comment by mraineri** [comment]
Approved 5/3

**Pull request merged**